ghost/i18n/locales/he/portal.json

```diff
@@ -0,0 +1,148 @@
+{
```

Comment:
Some lines seem to be missing in this file.  Intentional?

Reply:
Nope, it's probably a rebase oversight. I will fix it and push a new version of the portal file. Thanks for noticing!